### ISSUE TITLE
workdir cannot contain / when using qsub

### ISSUE THREAD

**Issue by KentonMurray** [levinboimtomer/JUST#9]
Here's the stacktrace. Removing the trailing slash from the workdir fixes the problem.

[kmurray4@crcfe01 il3]$ python ../../../../../../software/just/just.py basic_moses.just --workdir test_just/ --q '*@@nlp'
JUST: running: qsub -q *@@nlp -N s1.train.wd=test_just/ -v workdir=test_just/ test_just//s1.train.sh
qsub: ERROR! argument to -N option must not contain /

**Comment by levinboimtomer** [comment]
As the error message informs, the problem is with qsub's -N switch which assigns a name to the submitted job, and prohibits using "/".
How about replacing "/" with "\_" at line 206:
cmd_args.workdir.replace("/", "\_").

**Comment by KentonMurray** [comment]
Yeah, I think that's fine. This is only a minor bug ... you just get the / when you tab complete.
